pwndbg/commands/strings.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+import argparse
+import os
+from typing import List
+import pwndbg
+import pwndbg.commands
+import pwndbg.aglib.memory
+from pwndbg.commands import CommandCategory
+
+import re
+
+from pwndbg.lib.memory import Page
+
+parser = argparse.ArgumentParser(description="Extracts and displays ASCII strings from all readable memory pages of the debugged process.")
+
+parser.add_argument(
+    "-n",                 type=int,            default=4,    help="Minimum length of ASCII strings to include")
+parser.add_argument(
+    "mapping_name",       type=str, nargs="?", default=None, help="Mapping to search [e.g. libc]")
+parser.add_argument(
+    "--save-as",          type=str,            default=None, help="Sets the filename for the output of this command [e.g. --save-as='out.txt']")
+
+@pwndbg.commands.ArgparsedCommand(parser, command_name="strings", category=CommandCategory.LINUX)
```

Comment:
```suggestion
@pwndbg.commands.ArgparsedCommand(parser, category=CommandCategory.LINUX)
```
Unless I am mistaken this is only needed when command name differs from function name (e.g. `a_b` vs `a-b`). I believe `ArgparsedCommand` will automagically set command name to the `func.__name__`.